packages/compiler/server/serverlib.ts

```diff
@@ -465,6 +471,45 @@ export function createServer(host: ServerHost): Server {
     }
   }
 
+  function getNameandSymbolKind(node: Node): { name: string; kind: SymbolKind } | undefined {
+    switch (node.kind) {
+      case SyntaxKind.NamespaceStatement:
+        return { name: node.id.sv, kind: SymbolKind.Namespace };
+      case SyntaxKind.CadlScript:
+        return { name: node.id.sv, kind: SymbolKind.File };
+      case SyntaxKind.EnumStatement:
+        return { name: node.id.sv, kind: SymbolKind.Enum };
+      case SyntaxKind.InterfaceStatement:
+        return { name: node.id.sv, kind: SymbolKind.Interface };
+      case SyntaxKind.OperationStatement:
+        return { name: node.id.sv, kind: SymbolKind.Operator };
```

Comment:
Probably "function" or "method" is a better choice here. I'm guessing that's going to give a more familiar icon. We should try to stay consistent with getCompletionItemKind.